src/main/scala/zio/prelude/package.scala

```diff
@@ -61,6 +62,9 @@ package object prelude
   type DeriveInverse[F[_]]     = Derive[F, Inverse]
   type DeriveOrd[F[_]]         = Derive[F, Ord]
 
+  type Divariant[:=>[-_, +_]]   = Zivariant[({ type lambda[-R, +E, +A] = R :=> A })#lambda]
+  type Bicovariant[:=>[+_, +_]] = Zivariant[({ type lambda[-R, +E, +A] = E :=> A })#lambda]
```

Comment:
I would advise against using `:=>` as a type parameter in `Bicovariant`. `:=>` is (intentionally) similar to `=>` and thus hints that the first parameter is _contra_-variant, but that's not what happens here in `Bicovariant`, right?

Reply:
It bothers me too. Do you have a suggestion?

Reply:
Would something like plain `F` do?

Reply:
It is inconvenient to write all those [] that is why I prefer symbols rather than letters. Would `<=>` work?
https://github.com/zio/zio-prelude/pull/309/commits/1a39ee3b9000d4c3ef785e6fcf00a2f94dfd1b73

Reply:
`<=>` suggests `Equivalence` which is invariant :disappointed: 
https://github.com/zio/zio-prelude/blob/8074366be3814b16334bfd3f98aeee14b2dff174/src/main/scala/zio/prelude/package.scala#L34
`<*>`? `&&`? Or give up and use `F`?